src/includes/ConvertImageToByte.js

```diff
@@ -1,14 +1,19 @@
 const reader = new FileReader();
 const fileByteArray = [];
 
-function getConvertedImage(data) {
+/**
+ * Converts the image file to byte array.
+ * @param {file} image file.
+ * @return {Array} byte array consist of zeroes and ones.
```

Comment:
`consist of zeroes and ones`  is not needed as this is comment.

Reply:
done